microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,410 @@ ms.author: msedgedevrel
+```
+WebView2 Runtime (Stable) ->  Edge Beta -> Edge Dev -> Edge Canary
+```
 
-If a specific browser executable folder isn't specified, then the WebView2 will attempt to load a runtime from one of the known default locations.
+You can reverse the default search order by setting the `ChannelSearchKind` API to `LeastStable`, or by setting the `ChannelSearchKind` policy, registry key, or environment variable to `1`.  Reversing the search order makes the WebView2 loader search from least-stable (Canary) to most-stable:
 
-The default channel-search order is:
-1. The WebView2 Runtime.
-1. The Beta channel of Microsoft Edge.
-1. The Dev channel of Microsoft Edge.
-1. The Canary channel of Microsoft Edge.
+```
+Edge Canary -> Edge Dev -> Edge Beta -> WebView2 Runtime (Stable)
+```
 
-If you set the release channel preference to `1` by using a group policy, registry override, or environment variable, that will use the reverse of the default search order.
+This must be done before the WebView2 control is initialized.
 
 
-<!-- 1. Calling a function ============================================================== -->
-## Calling a function
+##### [API](#tab/api)
 
-If you want to make your application use a specific runtime by calling a function, complete the following steps.
+By default, the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property is `CoreWebView2ChannelSearchKind.MostStable` (an enum value).  Instead, reverse the search order by setting the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property to `CoreWebView2ChannelSearchKind.LeastStable`.
 
-### Win32\/C++
+.NET:
+* [CoreWebView2EnvironmentOptions.ChannelSearchKind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.channelsearchkind)
+* [CoreWebView2ChannelSearchKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2channelsearchkind)
 
-We'll use the [WebView2APISample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample) to demonstrate this procedure.
+WinRT:
+* [CoreWebView2EnvironmentOptions.ChannelSearchKind Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#channelsearchkind)
+* [CoreWebView2ChannelSearchKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2channelsearchkind)
 
-1. On your development machine, find the path containing the Microsoft Edge preview channel.  For example:
+Win32:
+* [ICoreWebView2EnvironmentOptions7::get_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7#get_channelsearchkind)
+* [ICoreWebView2EnvironmentOptions7::put_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7#put_channelsearchkind)
+* [COREWEBVIEW2_CHANNEL_SEARCH_KIND enum](/microsoft-edge/webview2/reference/win32/webview2-idl#corewebview2_channel_search_kind)
+<!-- todo: test API Ref links after April RelNotes is published ~Apr 22
```

Comment:
todo: test API Ref links after April RelNotes is published ~Apr 22
PR #3125 (Release Notes for April 2024 Release SDK & Prerelease SDK):
https://github.com/MicrosoftDocs/edge-developer/pull/3125
Live RelNotes to check for presence of April RelNotes:
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/

Reply:
ready to test now.  click links in rendered PR

Reply:
passed.